src/app/my-account/my-account/my-account.component.ts

```diff
@@ -43,7 +46,21 @@ export class MyAccountComponent implements OnInit {
   }
 
   removeAnnouncement(announcement: AccommodationOffer | MaterialAidOffer | TransportOffer): void {
-    console.log(announcement);
+    const dialogRef: MatDialogRef<ConfirmRemoveAdComponent> = this.dialog.open(ConfirmRemoveAdComponent, {
+      hasBackdrop: false,
+      width: '100%',
+      maxHeight: '450px',
+      maxWidth: '720px',
+    });
+
+    dialogRef.componentInstance.currentAnnouncement = announcement;
+
+    dialogRef.componentInstance.onClosed.pipe(take(1)).subscribe((confirmed: boolean) => {
+      dialogRef.close();
+      if (confirmed) {
+        // TODO: call to api
```

Comment:
Is API not ready yet?

Reply:
polaczenie z API bedzie developowane w oddzielnym zadaniu